scripts/local_testnet/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM rust:1.50.0 AS builder
+RUN apt-get update && apt-get install -y cmake
+
+RUN git clone https://github.com/sigp/lighthouse
+ARG PORTABLE
+ENV PORTABLE $PORTABLE
+RUN cd lighthouse && make
+RUN env
+RUN cd lighthouse && make install-lcli
```

Comment:
If we end up copying the lighthouse binaries from existing docker images we can delete all of this.